README.md

```diff
@@ -61,9 +61,38 @@ your jspm config file.
 `systemjs-hot-reloader` will automatically set `SystemJS.trace = true`, so you no longer
 need to set this manually, as with previous versions.
 
+### Worker usage
+Todays apps often use workers to move time intensive processing out of the browsers GUI thread. In such a case, use of hot-reloading is beneficial for the worker threads, too. Therefore `systemjs-hot-reloader` and [systemjs-hmr](https://github.com/alexisvincent/systemjs-hmr) can be used from workers.
+
+The following shows an example worker bootstrap script. It imports and configures `systemjs`, imports and connects the `systemjs-hot-reloader` and imports the implementation of the worker.
+```js
+importScripts('../node_modules/systemjs/dist/system.src.js', './system.config.js')
+
+System.import('systemjs-hot-reloader').then(function (connect) {
+    if (connect instanceof Function) {
+        connect()
+    }
+    System.import('WorkerImpl.js')
+})
+```
+
+Hints:
+* Using a single `systemjs` configuration (`system.config.js` in this case) for the GUI/main and the worker threads simplyfies the setup.
+* The worker is instantiated as usal: `new Worker("WorkerBootstrap.js")`
+
+### Circular dependencies / detection of entry points
```

Comment:
findEntries (systemjs-hmr@2.0.8) did not work in my case as the entry point is included from multiple other points of the application, too. In my case, the resulting entries list only contains modules that are not imported.

Sure, just change the order to your preferred variant!